node/directives/helper.ts

```diff
@@ -29,7 +29,14 @@ export const validateAdminToken = async (
       hasCurrentValidAdminToken = true
 
       if (authUser?.audience === 'admin') {
-        hasValidAdminToken = true
+        const hasAdminPermissions = await lm.getUserAdminPermissions(
+          authUser?.account,
```

Comment:
🟡 
It is not logical to call the LM if the account and the user id it is not populated. The result will always be an error.
If the authUser is null, the audience is not equal to `admin`, so using the nullish here just creates more cognitive load. Please remove authUser**?**

Reply:
Fixed